### ISSUE TITLE
ERROR - DB - version file not readable!

### ISSUE THREAD

**Issue by YuLanSky** [oschwengers/bakta#120]
Hi, 
Becaus a higher annotation accuracy is required, the prokka results is not enough,  I am looking forward the bakta annoation output.

However, may be due to  the network issues, my bakta is not working. 

1. bakta 1.4.2 was installed by bioconda, successfully;

2. $ bakta_db list --- is not working, 
Required database schema version: 3

HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /oschwengers/bakta/master/db-versions.json (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x10d30e3a0>: Failed to establish a new connection: [Errno 61] Connection refused'))
Traceback (most recent call last):
  File "/Users/chuchuli/opt/anaconda3/lib/python3.9/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/Users/chuchuli/opt/anaconda3/lib/python3.9/site-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/Users/chuchuli/opt/anaconda3/lib/python3.9/site-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 61] Connection refused

During handling of the above exception, another exception occurred: 
blablalbalba.................

3. $ bakta_db download --output <output-path> --- is not working, also "connection refused,Errno 61"

4. I download the DB manually
    $ wget https://zenodo.org/record/5961398/files/db.tar.gz --- OK!
    $ tar -xzf db.tar.gz --- OK!
    $ rm db.tar.gz --- OK!

5. $ amrfinder_update --force_update --database db/amrfinderplus-db/  --- is not working !!

Running: amrfinder_update --force_update --database db/amrfinderplus-db/
Looking up databases at https://ftp.ncbi.nlm.nih.gov/pathogen/Antimicrobial_resistance/AMRFinderPlus/database/

*** ERROR ***
Cannot create the root directory

HOSTNAME: ?
SHELL: /bin/zsh
PWD: /Users/chuchuli/Downloads
PATH: /Users/chuchuli/opt/anaconda3/bin:/Users/chuchuli/opt/anaconda3/condabin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/chuchuli/opt/sratoolkit.3.0.0-mac64/bin:/Users/chuchuli/opt/bowtie2-2.4.5-macos-x86_64:/usr/bin/javac
Progam name:  amrfinder_update
Command line: amrfinder_update --force_update --database db/amrfinderplus-db/

6. $ bakta --db /Volumes/ZHITAI/DB_bakta K-12.fa --- is not woring. 
ERROR: database version file (version.json) not readable!

Please tell me what should I do~~~~~~thx~~~~~

**Comment by oschwengers** [comment]
Hi @YuLanSky and thanks for reporting!
I just successfuly tested the download logic without any problems:
```
$ bakta_db download
fetch DB versions...
	... compatible DB versions: 2
download database: v3.1, 2022-02-03, DOI: 10.5281/zenodo.5961398, URL: https://zenodo.org/record/5961398/files/db.tar.gz...
|████████████████████████████████████████| 28593946/28593946 [100%] in 2:36:55.3 (3036.97/s)
	... done
check MD5 sum...
	...database file OK: 95a140c0c2a40153fdc9794e89860944
extract DB tarball: file=/home/oliver/tmp/db.tar.gz, output=/home/oliver/tmp
successfully downloaded Bakta database!
	version: 3.1
	DOI: 10.5281/zenodo.5961398
	path: /home/oliver/tmp/db
update AMRFinderPlus database...
	... done

Run Bakta using '--db /home/oliver/tmp/db' or set a BAKTA_DB environment variable: 'export BAKTA_DB=/home/oliver/tmp/db'
```

Therefore and due to your logs it seems that Bakta has a problem with opening an HTTPS connection to GitHub. This, however, should be a general problem with your connection. To test this, could you please try the following:
`wget https://raw.githubusercontent.com/oschwengers/bakta/master/db-versions.json` and check if this is working?

**Comment by oschwengers** [comment]
@YuLanSky *gentle ping* Could you download the file via the above mentioned command or even the entire database?

**Comment by YuLanSky** [comment]
Hello! Thank you for your reply. I am happy to tell you the problem I raised has been solved. It is indeed a network problem. The solution is to use a macbook to connect to a 5G phone hotspot first, and then run :

    $ amrfinder_update --force_update --database db/amrfinderplus-db/ --- OK!

Later, I deleted the database, tried to do the same operation again, and ran into the same problem.

    *** ERROR ***
    Cannot create the root directory

It is interesting why the network problem will report this error message :)

Thanks for your reply, Bakta is a very useful software!
